Tests/DependencyInjection/Compiler/LoggerChannelPassTest.php

```diff
@@ -51,7 +51,20 @@ public function testProcess()
             }
         }
 
-        $this->assertNotNull($container->getDefinition('monolog.logger.manualchan'));
+        $this->assertNotNull($container->getDefinition('monolog.logger.additional'));
+
+        // test below acceptable only for Symfony 4.2+
```

Comment:
make this a separate test, that get skipped

Reply:
Done